ast/location.go

```diff
@@ -188,7 +188,9 @@ func trimToLine(loc LocationRange, line int) LocationRange {
 // LineBeginning returns the part of a line directly before LocationRange
 // for example:
 // local x = foo()
-//           ^^^^^ <- LocationRange loc
+//
+//	^^^^^ <- LocationRange loc
```

Comment:
I have no approval powers here, but this comment got totally misformatted. Same many times below.

Reply:
Hey @rudo-thomas, I am using golang 1.19 in my environment. In 1.19 they updated how [godoc comments are formatted](https://tip.golang.org/doc/go1.19#go-doc). So I think that's where this change got pulled in from.

If we think that these formatting changes distract from the core of the PR, I'm happy to drop them. But anyone else that touches these files whilst using 1.19 is going to have similar issues.

Reply:
I'm saying these comments are now broken: The carets are no longer pointing where they used to – they don't make sense anymore.

The easiest would be to leave them unchanged. Possibly by using an earlier version of go fmt.

Reply:
Ah yes. I see what you are saying. Thanks for explaining. I think it should be fairly easy to fix this using godoc codeblocks. I've updated here: 2b535a5

Reply:
@rudo-thomas For the record, I really appreciate when people spot issues in PRs!